e-store/e-store/AppDependencies/DependencyManager.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  DependencyManager.swift
+//  e-store
+//
+//  Created by Ahmed Atef on 04/04/2023.
+//
+
+import Foundation
+import Swinject
+
+class DependencyManager {
```

Comment:
create new folder name Assembly and move storeAssembly on it 
create new folder name App and move DependencyManager on it